cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -275,7 +275,11 @@ public void move(Path source, Path targetDir) {
     try {
       Files.move(source, targetDir);
     } catch (IOException e) {
-      throw new IllegalStateException("Failed to move " + source + " to " + targetDir, e);
+      String fileType = Files.isSymbolicLink(source) ? "symlink" : isJunction(source) ? "junction" : Files.isDirectory(source) ? "directory" : "file";
+      if (this.context.getSystemInfo().isWindows() && fileType.equals("directory")) {
+        this.context.warning("On Windows, file operations could fail due to file locks. Please ensure the files in the moved directory are not in use.");
```

Comment:
I would rather include this to the exception message instead of an extra log.
Our plan is also to include some feature into IDEasy to support the user reporting an error as bug on github with the exception stacktrace included.
Further, it would be even more helpful to include this link as well:
https://github.com/devonfw/IDEasy/blob/main/documentation/how-to-unlock-files.adoc (should we rename the documentation page to `windows-file-lock.adoc`?)

Reply:
Done, the rename as well 🛩️